skills/lark-meeting/scripts/meeting_qa_bootstrap.py

```
@@ -0,0 +1,645 @@
+
+
+# ---- 基础设施 ----------------------------------------------------------------
+def log(msg: str) -> None:
+    """说明性文字统一走 stderr，永远不污染 stdout 的 Markdown 简报。"""
+    print(msg, file=sys.stderr, flush=True)
+
+
+def run_lark_json(args: list[str]) -> Optional[dict[str, Any]]:
+    """
+    执行一条 lark-cli 命令并解析其 JSON 输出。
+    lark-cli 在出错（returncode ≠ 0）时 JSON 会写到 stderr，所以需要同时捕获 stdout 和 stderr，
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)